client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -169,19 +170,152 @@ const InstructionsRenderer = ({
         commandsContent
     );
 
-    const assertions = [...pageContent.instructions.assertions.assertions];
-    const assertionsContent = parseListContent(assertions);
-
     const Heading = `h${headingLevel}`;
 
     return (
         <>
             <NumberedList>{allInstructionsContent}</NumberedList>
-            <Heading>{pageContent.instructions.assertions.header}</Heading>
-            {pageContent.instructions.assertions.description}
-            <NumberedList>{assertionsContent}</NumberedList>
             {settingsContent.length ? settingsContent : null}
 
+            {renderableContent.commands.map(
+                ({ id, settings, assertionExceptions = [] }, i) => {
+                    const settingsScreenText = isV2
+                        ? renderableContent.target.at.raw.settings[settings]
+                              ?.screenText ?? ''
+                        : null;
+
+                    let mustCount = 0;
+                    let shouldCount = 0;
+                    let mayCount = 0;
+
+                    let assertions = [...renderableContent.assertions];
```

Comment:
I noticed in the alert example the should assertions were coming before the must assertions, which makes me think that sorting this array might be needed.

<img width="773" alt="Screenshot 2024-03-05 at 5 05 33 PM" src="https://github.com/w3c/aria-at-app/assets/5255858/f2a05c97-7c78-458f-b9a6-795c84d5fedd">